sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -470,7 +470,7 @@ internal class RadarLocationManager(
         val ignoreSync =
             lastSentAt == 0L || this.callbacks.count() > 0 || justStopped || replayed
         val now = System.currentTimeMillis()
-        val lastSyncInterval = now - lastSentAt
+        val lastSyncInterval = (now - lastSentAt) / 1000L
```

Comment:
Converts milliseconds to seconds, which can be compared to `desiredSyncInterval` in seconds on line 481

Reply:
could be helpful for future readers to have this as a comment? or above the conditional on L481?

Reply:
or you could just call `lastSyncInterval` -> `lastSyncIntervalSeconds`